### DISCUSSION TITLE
how to get badges on github and will these profile really make a difference in job hunt

### DISCUSSION THREAD

**Discussion by Varchasavkr** [community/community#144074] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

how to get badges on github and will these profile really make a difference in job hunt

**Comment by dalvarez2596** [comment, marked as answer]
Hi @Varchasavkr , for most GitHub achievements, you need to complete certain criteria. You can take a look at this [guide](https://drknzz.github.io/GitHub-Achievements/), which is a great resource, and it shows you how to earn them. Some of the achievements highlight your contributions to the open-source community and how involved you are. They also look cool on your profile 😎, so they can help make your profile more attractive. I also suggest working on side projects and creating your own portfolio or GitHub Pages to showcase the projects you've worked on

Refer to this about the github achievements: 
[Github Achievements](https://githubachievements.com/#quickdraw-section)
